netdata-installer.sh

```diff
@@ -550,17 +551,18 @@ bundle_libmosquitto
 # -----------------------------------------------------------------------------
 
 build_libwebsockets() {
+  CFLAGS="-fPIC"
   pushd "${1}" > /dev/null || exit 1
   if [ "$(uname)" = "Darwin" ] && [ -d /usr/local/opt/openssl ]; then
-    run env CFLAGS= CXXFLAGS= LDFLAGS= cmake \
+    run env CFLAGS="$CFLAGS" CXXFLAGS= LDFLAGS= cmake \
```

Comment:
Since this used to be `CFLAGS=` because there were some kind of issues when `CFLAGS` were not cleared, and since `libmosquitto` explicitly sets `-fPIC` in its build system, and since we want to build `libmosquitto` and `libwebsockets` together, isn't it safer to do `CFLAGS=-fPIC` instead of `CFLAGS="$CFLAGS"? This way we both clear `CFLAGS` and force `-fPIC` for all cloud libraries. In case of a build issue, then netdata is safe to be built on its own afterwards without cloud support.

Reply:
Unless I'm mistaken, the proposed change is semantically equivalent to what you're suggesting, just not as obvious as it could be.

Reply:
Whoops, you're right, I missed the `CFLAGS="-fPIC"` :D

Reply:
Perhaps a change of name (`SAFE_CFLAGS` maybe?) might be in order.

Reply:
And I _thought_ I was blind 😂